packages/cli/src/services/redis/RedisServiceCommands.ts

```diff
@@ -12,7 +12,7 @@ export type RedisServiceCommand =
  * @field payload: Optional arguments to be sent with the command.
  */
 type RedisServiceBaseCommand = {
-	senderId?: string;
+	senderId: string;
```

Comment:
should we perhaps remove this from here? this property gives an impression that whenever we are publishing a command, we can include a `senderId`, which isn't true. not to mention, we have to do `Omit<RedisServiceCommandObject, 'senderId'>` to block people from adding a `senderId` in the payload.

Reply:
I could go the inverse and remove it here and have another interface with added senderId